SQL_module/SubjectForm/tableElectives.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE electives (
```

Comment:
Nice one! Would be even better if you add script for creating the DB itself. Hint: phpmyadmin has "export" functionality.

Reply:
Hint 2: select the DB that you are using first. Then go to export. Then select custom export. Then select "Add CREATE DATABASE / USE statement".